app/src/main/java/org/simple/clinic/home/HomeScreenEvent.kt

```diff
@@ -5,6 +5,7 @@ import org.simple.clinic.patient.Patient
 import org.simple.clinic.patient.businessid.Identifier
```

Comment:
When we don't find the patient we open the search screen. Can we update the commit message to reflect that?

Reply:
Oh wait, I thought I pushed that change. My bad will fix it 😅